code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgCreateEventFragment.java

```diff
@@ -0,0 +1,22 @@
+package com.example.eventlinkqr;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+/**
+ * this class takes care of taking in the input for a new event and adding it to the data
+ */
+public class OrgCreateEventFragment extends Fragment {
+
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
If the is an override, then it should probably have the `@Override` on it. Otherwise, it should have a javadoc comment